mslice/scripting/helperfunctions.py

```diff
@@ -0,0 +1,217 @@
+from datetime import datetime
+from mslice.cli.helperfunctions import _function_to_intensity
+from mslice.models.labels import get_recoil_key
+PACKAGES = {
+    'mslice.cli': 'mc',
+    'mslice.plotting.pyplot': 'plt',
+    'matplotlib.colors': 'colors',
+    'numpy': 'np',
+}
+
+
+def cleanup(script_lines):
+    for line in script_lines[-1:]:
+        if line.startswith('Load'):
+            index = script_lines.index(line)
+            return script_lines[index:]
+
+
+def header(plot_handler):
+    from mslice.plotting.plot_window.cut_plot import CutPlot
+    from mslice.plotting.plot_window.slice_plot import SlicePlot
+    statements = ['# Python Script Generated by Mslice on {}\n'.format(datetime.now().replace(microsecond=0))]
+
+    for package in PACKAGES:
+        if 'colors' in package:
+            if isinstance(plot_handler, SlicePlot) and plot_handler.colorbar_log is not True or \
+                    isinstance(plot_handler, CutPlot):
+                continue
+        elif 'numpy' in package:
+            if isinstance(plot_handler, CutPlot) and (plot_handler.x_log is not True or plot_handler.y_log is not True):
+                continue
+            elif isinstance(plot_handler, SlicePlot):
+                continue
+        statements.append('import {} as {}\n'.format(package, PACKAGES[package]))
+
+    return statements
+
+
+def add_header(script_lines, plot_handler):
+    for i, statement in enumerate(header(plot_handler)):
+        script_lines.insert(i, statement)
+
+
+def add_plot_statements(script_lines, plot_handler):
+    from mslice.plotting.plot_window.slice_plot import SlicePlot
+    from mslice.plotting.plot_window.cut_plot import CutPlot
+
+    add_header(script_lines, plot_handler)
+
+    line_no = len(script_lines)
+    script_lines.insert(line_no - 1, '\n')
+    script_lines[line_no] = 'ws = mc.{}\n'.format(script_lines[line_no])
+
+    if plot_handler is not None:
+        if isinstance(plot_handler, SlicePlot):
+            add_slice_plot_statements(script_lines, plot_handler)
+            add_overplot_statements(script_lines, plot_handler)
+        elif isinstance(plot_handler, CutPlot):
+            add_cut_plot_statements(script_lines, plot_handler)
+
+        script_lines.append('mc.Show()\n')
+
+    return script_lines
+
+
+def add_slice_plot_statements(script_lines, plot_handler):
+    slice = plot_handler._slice_plotter_presenter._slice_cache[plot_handler.ws_name]
+    momentum_axis = str(slice.momentum_axis)
+    energy_axis = str(slice.energy_axis)
+    norm = slice.norm.vmin == 0.0 and slice.norm.vmax == 1.0
+
+    script_lines.append('slice_ws = mc.Slice(ws, Axis1=\'{}\', Axis2=\'{}\', NormToOne={})\n\n'.format(
```

Comment:
Just a note that in Python you can mix `"` and `'` for strings so here you could do
```script_lines.append("slice_ws = mc.Slice(ws, Axis1='{}', Axis2='{}', NormToOne={})\n\n".format(```
to avoid having to escape the quotes `'`, which I think would improve readability.

This is true in multiple places in this file I think.